deployment/scripts/.env

```diff
@@ -0,0 +1,16 @@
+# k8s envs
+k8s_provider="eks"
+k8s_namespace="fuel-core"
+
+# Faucet envs
+fuel_committer_image_repository="ghcr.io/fuellabs/fuel-block-committer"
+fuel_committer_image_tag="latest"
```

Comment:
never use `latest` for specifying image as per best practices

Reply:
this is the base that's meant to be overwritten in by env overlays, so the values here are somewhat arbitrary. But I can set them to dummy values to ensure it doesn't accidentally succeed in an unexpected way.

Reply:
see https://github.com/FuelLabs/fuel-block-committer/pull/35/files#r1286509748